assets/js/atomic/blocks/product-elements/image/block.js

```diff
@@ -29,10 +29,11 @@ import './style.scss';
  * @param {string} [props.saleBadgeAlign] How should the sale badge be aligned if displayed.
  * @return {*} The component.
  */
-const Block = ( {
+export const Block = ( {
 	className,
 	imageSizing = 'full-size',
-	productLink = true,
+	// @todo Rename productLink to showProductLink to better reflect it's a Boolean
```

Comment:
Do we still need this todo?

Reply:
I think this should still be handled at project level, as this prop name is used in other parts as well. For example, the component receives `showSaleBadge` and `productLink`. Both are booleans, and trigger a different rendered output. But the shape of the prop name is different and I found that confusing. With the current name, `productLink` appears to be a string, an actual link value.

Do you think I should update the comment with more details?

Reply:
I would remove `@todo` here and just create a simple issue and tag it as a refactor. You can quickly explain what is it about there.

Reply:
[Issue](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3756) created